rosbag2_transport/include/rosbag2_transport/player.hpp

```diff
@@ -74,6 +74,10 @@ class Player : public rclcpp::Node
   /// delete_on_play_message_callback.
   using callback_handle_t = uint64_t;
 
+  /// \brief Utility type for a pair of rosbag2_cpp::Reader and rosbag2_storage::StorageOptions.
+  using ReaderStorageOptionsPair = \
```

Comment:
Suggesting to rename for consistency since this is an inner-class definition.
```suggestion
  using reader_storage_options_pair_t = \
```

Reply:
renamed in ca23e7fae501e6a2950f529ef0dafa2f37dd1f26